mantaray/commands.py

```diff
@@ -151,6 +151,10 @@ def msg_nickserv(view: View, core: IrcCore, message: str) -> None:
     def msg_memoserv(view: View, core: IrcCore, message: str) -> None:
         return msg(view, core, "MemoServ", message)
 
+    @add_command("/kick <nick> [<reason>]")
+    def kick(view: View, core: IrcCore, nick: str, reason: str) -> None:
```

Comment:
Allow it to be None, and give it a default value of None
```suggestion
    def kick(view: View, core: IrcCore, nick: str, reason: str | None = None) -> None:
```